src/renderer/i18n/locales/en-US.js

```diff
@@ -506,23 +506,26 @@ export default {
     DOWNLOAD: 'Download now',
     TITLE: 'Permissions',
     PERMISSIONS: {
-      COMPONENTS: 'Load in custom components',
-      ROUTES: 'Loads additional routes into the Desktop Wallet',
-      MENU_ITEMS: 'Allows adding custom menu items to the Desktop Wallet sidebar',
+      ALERTS: 'Allows access to the Desktop Wallet alerts',
+      AUDIO: 'Allows access to play audio from within the Desktop Wallet',
       AVATARS: 'Plugin contains custom components',
-      WALLET_TABS: 'Allows showing an additional tab/page on the Wallet screen',
-      THEMES: 'Allows loading additional custom themes for the Desktop Wallet',
-      UI_COMPONENTS: 'Allows access to the standard Desktop Wallet components used throughout',
-      WEBFRAME: 'Allows showing remote URL pages within a frame',
+      COMPONENTS: 'Allows to load custom components',
       EVENTS: 'Allows access to the Desktop Wallet events',
-      AUDIO: 'Allows access to play audio from within the Desktop Wallet',
-      ALERTS: 'Allows access to the Desktop Wallet alerts',
-      MESSAGING: 'Allows WebFrame to have access to a one-way messaging system',
-      STORAGE: 'Allows storing data within the Desktop Wallet, using key-value pairs',
       HTTP: 'Allows performing external web requests',
+      MENU_ITEMS: 'Allows adding custom menu items to the Desktop Wallet sidebar',
+      MESSAGING: 'Allows WebFrame to have access to a one-way messaging system',
       PEER_CURRENT: 'Allows access to the currently connected peer',
+      PUBLIC: 'Allows to navigate to wallet routes and provides access to the Font Awesome icon set',
+      PROFILE_ALL: 'Allows access to all available profiles',
       PROFILE_CURRENT: 'Allows access to the currently active profile',
-      PROFILE_ALL: 'Allows access to all available profiles'
+      ROUTES: 'Allows to load additional routes into the Desktop Wallet',
+      STORAGE: 'Allows storing data within the Desktop Wallet, using key-value pairs',
+      THEMES: 'Allows loading additional custom themes for the Desktop Wallet',
+      TIMERS: 'Allows using timeouts and intervals',
+      UI_COMPONENTS: 'Allows access to the standard Desktop Wallet components used throughout',
+      WALLET_TABS: 'Allows showing an additional tab/page on the Wallet screen',
+      WEBFRAME: 'Allows showing remote URL pages within a frame',
+      WEBSOCKET: 'Allows to connect to websockets'
```

Comment:
"Allows to" doesn't sound natural. I recommend changing the phrases to something like:

"Allows loading custom components"
"Allows navigation to wallet routes and provides access to the Font Awesome icon set"
"Allows loading additional routes into the Desktop Wallet"
"Allows connections to websockets" 

Also:
"Allows WebFrames access to a one-way messaging system"

Reply:
update, thanks